data/pages/faq-grantees.mdx

```diff
@@ -0,0 +1,224 @@
+
+Not necessarily. We review all reports, but this work is mostly done by a team
+of volunteers with technical expertise in the area of your project. Given the
+nature of this volunteer work, there is not a set schedule for this review to
+happen, so please do not expect feedback for every report, immediate or not.
+
+We will contact you if we have any questions or concerns about your report.
+
+### What happens if the deliverables, milestones, or timelines proposed in the application are changed or not delivered?
+
+Software development is inherently unpredictable. If you make changes to your
+project, explain the reasoning clearly in your progress reports and keep on
+building!
+
+### Is it okay to work on something that isn’t part of the grant agreement or even pivot to another project?
+
+The short answer is yes, within reason. Your work must be aligned with [our
+mission](/mission) and meet our criteria of being free and open-source within
+the Bitcoin and/or Nostr ecosystems.
+
+[Reach out to us][c] if you'd like to explore this option further.
+
+### I wasn’t able to work on my project as much as I expected, what should I do?
+
+[Reach out to us][c] to discuss the best path forward as soon as possible.
+
+### Can I pause my grant?
+
+Yes. Let us know the duration of the pause before taking a break.
+
+### Do I need to track hours?
+
+No. Our expectation is that grantees will work on deliverables and milestones
+listed in their grant agreement in an honest manner and on a best-effort basis.
+Grantees have the flexibility to manage their workload as they see fit.
+
+Work done should be reflected in each progress report. Just be honest.
+
+Taking a few days, weekends, and evenings off is perfectly fine and encouraged.
+We shouldn’t have to write this, but past experience forces us to be explicit
+about it: **Please do not sacrifice your health and well-being.** We encourage
+our grantees to do good work that is aligned with our mission, but we also
+encourage you to live a balanced life. Do not burn yourself to a crisp.
+
+As mentioned above, if you need to pause a grant to take extended time off, let
+us know as soon as possible and we’ll do our best to find a solution that works
+for you and us.
+
+### How can I best ensure that OpenSats is getting a good return on investment?
+
+Some grantees feel pressure to provide ROI to OpenSats. Do not think about it
+this way - we are not a VC fund; we are not making investments. We expect you to
+do good, honest work.
+
+We select people who are aligned with our mission and values, and we trust that
+the rest will fall into place. Our expectation is that you're acting in good
+faith.
+
+### Does my grant renew automatically?
+
+No, each grant has a set end date listed in the grant agreement. If you would
+like to apply for another grant, consider applying 60-90 days before the end of
+your current grant in order to give us enough time to evaluate and process your
+application.
```

Comment:
Might be worthwhile to clarify that this is not the case for LTS grants (or if it is, then that).

Reply:
LTS grants are two years, but this applies to LTS grants as well. Applying 60-90 days before the end of the grant is ideal for all grantees that would like to apply for further funding.

Reply:
@0xB10C initially a few LTS grants (yours included) were approved for 12 months. A few months later, the board decided to default all existing and new LTS grants to 24 months. So any LTS grant that had been approved for only 12 months (yours included) will be auto-renewed for an additional 12-month period. At the end of the 24-month LTS cycle, the Q&A above will apply equally to all LTS grants.